packages/components/src/components/FileUploader/FileUploader.tsx

```diff
@@ -187,7 +193,7 @@ export const FileUploader = (props: FileUploaderProps) => {
             id={inputId}
             data-testid="file-uploader-input"
           />
-          Dra og slipp filer her eller{' '}
+          {dropAreaLabel}
           <VisuallyHidden as="span">
             velg fil med påfølgende knapp
```

Comment:
Litt usikker på håndtering av den usynlige teksten. Akkurat nå henger den tett sammen med den synlige, og det kommer ikke tydelig fram i `dropAreaLabel`-propen at  de henger sammen. Kan bli kluss med UU 🤔

Reply:
Mhm. Tenker isåfall at du kunne utvide prop-beskrivelse på linje 72 i `FileUploader.tsx`, slik at den nevner at teksten skal henge med den usynlige. Så er vi good 👍